Sources/FTPropertyWrappers/UserDefault.swift

```diff
@@ -19,6 +37,12 @@ public final class DefaultsStore<Value: Codable> {
         self.defaultValue = defaultValue
     }
 
+    /// Getter and setter for enclosed property. Property itself is not stored in this class and is always returned and
+    /// stored from/into UserDefaults. This computed property attempts to store/load type as if it was a complex
+    /// propety first. In case it is not encodable/decodable, this getter attemts to retrieve it it's raw form and cast it
```

Comment:
I am not sure what you wanted to describe once again.

```suggestion
    /// propety first. In case it is not encodable/decodable, this getter attemts to retrieve if its in raw form and cast it
```